examples/node/MarketDataSubscription.js

```diff
@@ -42,16 +42,19 @@ socket.on('err', function (data) {
     console.log(data);
 });
 
-socket.on('subscribed', function () {
+socket.on('subscribed', function (correlationIds) {
     console.log('Subscribed');
+    console.log('Correlation Ids: ' + correlationIds);
 });
 
-socket.on('unsubscribed', function () {
+socket.on('unsubscribed', function (correlationIds) {
     console.log('Unsubscribed');
+    console.log('Correlation Ids: ' + correlationIds);
 });
 
-socket.on('unsubscribed all', function () {
+socket.on('unsubscribed all', function (correlationIds) {
```

Comment:
Now that we have correlation id's, I wonder if 'unsubcribed all' is necessary to echo back to the client anymore.  @ericlu88, what do you think?